WebDriverAgentLib/Categories/XCUIElement+FBPickerWheel.h

```diff
@@ -16,20 +16,30 @@ NS_ASSUME_NONNULL_BEGIN
 /**
  Selects the next available option in Picker Wheel
  
+ @param offset the value in range [0.01, 0.5]. It defines how far from picker
+   wheel's center the click should happen. The actual distance is culculated by
+   multiplying this value to the actual picker wheel height. Too small offset value
+   may not change the picker wheel value and too high value may cause the wheel to switch
+   two or more values at once. Usually the optimal value is located in range [0.15, 0.3]
  @param error returns error object if there was an error while selecting the
    next picker value
  @return YES if the current option has been successfully switched. Otherwise NO
  */
-- (BOOL)fb_selectNextOptionWithError:(NSError **)error;
+- (BOOL)fb_selectNextOptionWithOffset:(CGFloat)offset error:(NSError **)error;
 
 /**
  Selects the previous available option in Picker Wheel
  
- @param error returns error object if there was an error while selecting the 
+ @param offset the value in range [0.01, 0.5]. It defines how far from picker
+   wheel's center the click should happen. The actual distance is culculated by
+   multiplying this value to the actual picker wheel height. Too small offset value
+   may not change the picker wheel value and too high value may cause the wheel to switch
+   two or more values at once. Usually the optimal value is located in range [0.15, 0.3]
+ @param error returns error object if there was an error while selecting the
    previous picker value
  @return YES if the current option has been successfully switched. Otherwise NO
  */
-- (BOOL)fb_selectPreviousOptionWithError:(NSError **)error;
```

Comment:
I would be happy to add all the necessary details if you can provide a template for it